### DISCUSSION TITLE
Possible to use trusted certificates?

### DISCUSSION THREAD

**Discussion by solvistadev** [docusealco/docuseal#7] [Q&A] [answered]
Is it possible to use my verified certificates from a trusted CA so that I get 'green checks' in eg adobe reader?

**Comment by AlexBTurchyn** [comment, marked as answer]
@sanhup Currently it's not possible - but this will be added eventually, will let you know when this feature is released

**Reply by solvistadev** [reply]
Ok nice, thank you!

**Reply by AlexBTurchyn** [reply]
@sanhup We have recently added the ability to upload user's certificates. You can use [this page](https://docuseal.co/settings/esign) to upload your trusted certificate.
